pom.xml

```diff
@@ -690,6 +696,9 @@ under the License.
                   <ignoredScopes>
                     <ignoredScope>test</ignoredScope>
                   </ignoredScopes>
+                  <excludes>
+                    <exclude>org.jline:jline</exclude>
+                  </excludes>
```

Comment:
Why? Which JDK is required?
And what when user have an older JDK?

Reply:
The reason is https://github.com/jline/jline3/issues/924